OamLog+config.json

```diff
@@ -978,10 +978,10 @@
                     "remote-protocol": "http",
                     "remote-address": {
                       "ip-address": {
-                        "ipv-4-address": "10.118.125.157"
```

Comment:
old-release ip-address to be based on release-number 1.0.0, where it would be 1.1.3.4 and 3004

Reply:
is changed now